language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ObjectBodiedNodeContextProvider.java

```diff
@@ -16,10 +16,12 @@
 package org.ballerinalang.langserver.completions.providers.context;
 
 import io.ballerina.compiler.api.symbols.Symbol;
+import io.ballerina.compiler.syntax.tree.FunctionDefinitionNode;
```

Comment:
Shall we increase the priority of the init function as well. Refer the following where I don't see the snippet at the top
<img width="961" alt="Screenshot 2022-02-07 at 12 32 37" src="https://user-images.githubusercontent.com/1329674/152740494-101b03c3-66fa-48fa-9b30-65fe8f014494.png">

Reply:
Done 👍 
<img width="539" alt="Screenshot 2022-02-07 at 16 06 51" src="https://user-images.githubusercontent.com/27485094/152772139-c8ea2d62-437c-4862-a73f-1b7dfd03e5ad.png">